### ISSUE TITLE
The question of learning rate

### ISSUE THREAD

**Issue by JayChan-USTC** [tianweiy/CenterPoint#238]
HI，author!
The model learning rate is defined as follows:
"
lr_scheduler = _create_learning_rate_scheduler(
            optimizer, cfg.lr_config, total_steps)

def step(self, step):
      lrs, moms = [], []
       for start, end, func in self.lr_phases:
           if step >= start:
               lrs.append(func((step - start) / (end - start)))
       if len(lrs) > 0:
           self.optimizer.lr = lrs[-1]    
"

As the number of learning steps increases, the shape of the learning rate value is like a cosine function.
The value first rises, when the number of steps reaches 6520, the value starts to decrease.

1.Why design such a learning rate update function instead of designing a larger learning rate at the beginning, which gradually decreases as the number of learning steps increases.

2.Another question is why 6520 is chosen as the turning step for learning rate value, and does it have any special meaning?

**Comment by tianweiy** [comment]
> 1.Why design such a learning rate update function instead of designing a larger learning rate at the beginning, which gradually decreases as the number of learning steps increases.

I think this will also work. But as everyone uses this one cycle policy (second, det3d, openpcdet), I simply follow them.

> 2.Another question is why 6520 is chosen as the turning step for learning rate value, and does it have any special meaning?

6520 is computed using the pc_start parameter (it is 0.4 of the training iterations) https://github.com/tianweiy/CenterPoint/blob/952f14b694970cb667d8e3fa00ab5ae0936fa164/configs/mvp/nusc_centerpoint_pp_fix_bn_z_scale.py#L217 

Again, I follow the schedule from other repos.

**Comment by JayChan-USTC** [comment]
get it! Thank you.
